cromwell.example.backends/AWS.conf

```diff
@@ -9,60 +9,49 @@
 # Documentation
 # https://cromwell.readthedocs.io/en/stable/tutorials/AwsBatch101/
 
-aws {
-
-  application-name = "cromwell"
-
-  auths = [
-    {
-      name = "default"
-      scheme = "default"
-    }
-  ]
-
-  region = "<your-region>"
-}
-
-engine {
-  filesystems {
-    s3 {
-      auth = "default"
+  aws {
+    application-name = "cromwell"
+    auths = [{
+        name = "default"
+        scheme = "default"
+    }]
+    region = "${AWS::Region}"
+  }
+  database {
+    profile = "slick.jdbc.MySQLProfile$"
+    db {
+      driver = "com.mysql.cj.jdbc.Driver"
+      url = "jdbc:mysql://${DBAddress}/cromwell?rewriteBatchedStatements=true&useSSL=false"
+      user = "${DBUsername}"
+      password = "${DBPassword}"
+      connectionTimeout = 5000
```

Comment:
This database section seems not particularly relevant to the `AWS` backend example conf?